primitives/consensus/babe/src/digests.rs

```diff
@@ -88,6 +119,9 @@ pub enum RawPreDigest<VRFOutput=schnorrkel::RawVRFOutput, VRFProof=schnorrkel::R
 	/// A secondary deterministic slot assignment.
 	#[codec(index = "2")]
 	Secondary(SecondaryPreDigest),
+	/// A secondary deterministic slot assignment with VRF outputs.
+	#[codec(index = "3")]
+	SecondaryVRF(RawSecondaryVRFPreDigest<VRFOutput, VRFProof>),
```

Comment:
Since I think this will become the default why not name this one `Secondary` and rename the existing one to `SecondaryPlain` (or some other better name).

Reply:
How about just naming it `SecondaryPlain` and `SecondaryVRF`? Just thinking that will make it clearer.

Reply:
Yep good idea! :+1: